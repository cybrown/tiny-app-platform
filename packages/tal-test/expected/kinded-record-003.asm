main()
  entry:
    FunctionRef       name: func_0
func_0()
  entry:
    Local             View
    Local             Text
    FunctionRef       name: func_1
    Literal           2
    MakeArray
    Literal           "boolPropTrue1"
    Literal           true
    Literal           "boolPropTrue2"
    Literal           true
    Literal           "boolPropFalse"
    Literal           false
    Literal           "propA"
    Literal           "a"
    Literal           "propAAA"
    Literal           "aaa"
    Literal           5
    MakeRecord
    Kinded
func_1()
  entry:
    Local             View
    Literal           0
    MakeArray
    Literal           "boolPropFalse"
    Literal           false
    Literal           1
    MakeRecord
    Kinded
