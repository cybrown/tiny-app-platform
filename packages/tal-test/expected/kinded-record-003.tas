View {
  --boolPropTrue1
  --boolPropTrue2
  -!boolPropFalse
  propA:   "a"
  propAAA: "aaa"

  Text
  View { -!boolPropFalse }
}
