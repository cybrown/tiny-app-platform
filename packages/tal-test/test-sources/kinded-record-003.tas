View {
  --boolPropTrue1
  Text
  --boolPropTrue2
  -!boolPropFalse
  propA: "a"
  propAAA: "aaa"

  View {
    -!boolPropFalse
  }
}
