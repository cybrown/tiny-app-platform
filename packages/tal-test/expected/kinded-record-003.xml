<program>
  <kindedrecord>
    <local />
    <local />
    <kindedrecord>
      <local />
      <kindedrecordentry>
        <literal />
      </kindedrecordentry>
    </kindedrecord>
    <kindedrecordentry>
      <literal />
    </kindedrecordentry>
    <kindedrecordentry>
      <literal />
    </kindedrecordentry>
    <kindedrecordentry>
      <literal />
    </kindedrecordentry>
    <kindedrecordentry>
      <literal />
    </kindedrecordentry>
    <kindedrecordentry>
      <literal />
    </kindedrecordentry>
  </kindedrecord>
</program>
